server/.eslintrc.json

```diff
@@ -33,49 +29,44 @@
         "camelcase": "off",
         "require-await": "error",
         "@typescript-eslint/no-dupe-class-members": ["error"],
-        "@typescript-eslint/explicit-member-accessibility": [
-            "error",
-            { "accessibility": "explicit" }
-        ],
+        "@typescript-eslint/explicit-member-accessibility": ["error", { "accessibility": "explicit" }],
         "@typescript-eslint/explicit-module-boundary-types": ["error"],
-        "@typescript-eslint/no-inferrable-types": [
+        "@typescript-eslint/no-inferrable-types": ["error", { "ignoreParameters": true }],
+        "@typescript-eslint/naming-convention": [
             "error",
-            { "ignoreParameters": true }
```

Comment:
Why was this removed?

Reply:
looks like git diff is just being weird. ignoreParameters was never under naming-convention https://github.com/NachoToast/SudoCluedo/blob/main/server/.eslintrc.json#L43